ENGAGEHF/ENGAGEHFStandard.swift

```diff
@@ -122,15 +146,15 @@ actor ENGAGEHFStandard: Standard, EnvironmentAccessible, OnboardingConstraint, A
         var collectionBucket: String? {
             switch type {
             case HKQuantityType(.bodyMass):
-                return "bodyWeightObservations"
+                return CollectionID.bodyWeightObservations.rawValue
             case HKQuantityType(.bodyMassIndex):
                 return "bodyMassIndexObservations"
```

Comment:
Might make sense to add that to the `CollectionID` type, even if we don't really use that collection everywhere or just return nil here.

Reply:
I went ahead and set it to nil, since we set up a listener to every collection in CollectionID. This would be fine if we wanted to listen for BMI and whatnot, but for now I figured we can ignore this case. Definitely me know if it would instead be better to choose which collections to listen to without iterating through every case of CollectionID, however.